android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -161,8 +161,22 @@ public void updateSurfaceView(@ViewType.ViewType int viewType) {
         }
     }
 
+    private void hideShutterView() {
+        shutterView.setVisibility(INVISIBLE);
+        ((SurfaceView)surfaceView).setAlpha(1);
```

Comment:
What about when the user configures the texture view ?

Reply:
this was cast only - it's removed so it should be fine now